sdk/identity/Azure.Identity/README.md

```diff
@@ -107,6 +107,8 @@ var credential = new DefaultAzureCredential(new DefaultAzureCredentialOptions {
 var blobClient = new BlobClient(new Uri("https://myaccount.blob.core.windows.net/mycontainer/myblob"), credential);
 ```
 
+In addition to configuring the `ManagedIdentityClientId` via code, it can also be set using the `AZURE_CLIENT_ID` environment variable. These two approaches are equivalent.
```

Comment:
I think perhaps we should qualify the last  sentence a bit, to something like:

>These two approaches are equivalent, when using the `DefaultAzureCredentiall`

This is because only `DefaultAzureCredential` honors this environment variable, the `ManagedIdentityCredential` will not.  What do you think?